### PR TITLE
Improves fetching speed

### PR DESCRIPTION
**Pull request by MarcosFleury**

### CODE REVIEW

File: Jobs.Fetcher.YouTube/Helpers/ApiDataFetcher.cs [hunk 1]
```diff
@@ -209,7 +209,6 @@ public static class ApiDataFetcher {
             catch {
                 //logger.Information("Could not get Subscriber Views");
```
Comment (moraesvic) [inline Jobs.Fetcher.YouTube/Helpers/ApiDataFetcher.cs:210]: Error is not getting logged

File: Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs [hunk 2]
```diff
@@ -77,8 +77,6 @@ public class ReprocessDailyVideoMetricsQuery : YoutubeFetcher {
                 if (item.Lifetime > 0) {
                     var ratio = Math.Abs((double) item.Lifetime - item.Total) / ((double) item.Lifetime);
                     if (forceFetch || Math.Abs(ratio) > comparisonThreshold && item.Lifetime > comparisonMinLimit) {
-                        Console.WriteLine("Reprocessing video {0}: daily views {1} from {4} to {5}, lifetime views {2} at {6} and ratio {3}"
-                                          , item.Id.VideoId, item.Total, item.Lifetime, ratio, item.DailyStart, item.DailyEnd, item.LifetimeDate);
                         Logger.Information("Reprocessing video {0}: daily views {1} from {4} to {5}, lifetime views {2} at {6} and ratio {3}"
```
Comment (moraesvic) [inline Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs:80]:
```suggestion
                        Logger.Information("Reprocessing video {0}: daily views {1} from {2} to {3}, lifetime views {4} at {5} and ratio {6}"
```

File: Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs [hunk 3]
```diff
@@ -77,8 +77,6 @@ public class ReprocessDailyVideoMetricsQuery : YoutubeFetcher {
...
                                            , item.Id.VideoId, item.Total, item.Lifetime, ratio, item.DailyStart, item.DailyEnd, item.LifetimeDate);
```
Comment (moraesvic) [inline Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs:81]:
```suggestion
                                           , item.Id.VideoId, item.Total, item.DailyStart, item.DailyEnd, item.LifetimeDate, item.Lifetime, ratio);
```

Comment (moraesvic) [reply Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs:81]: I think I still prefer the format `$"Reprocessing video {item.Id.VideoId}" ... `, but maybe there is some limitation I'm not aware of

**Review by moraesvic — CHANGES_REQUESTED** [review]

File: Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs (hunk 3 above)
Comment (MarcosFleury) [reply Jobs.Fetcher.YouTube/YoutubeFetcherJobs.cs]: I think all this information is not really that necessary so I just left the information of which video had its metrics reprocessed

**Review by moraesvic — APPROVED** [review]

**Pull request merged**